easybuild/easyconfigs/c/casacore/casacore-3.5.0-foss-2023b.eb

```
@@ -7,16 +7,30 @@ homepage = 'https://github.com/casacore/casacore'
 description = """A suite of C++ libraries for radio astronomy data processing.
 The ephemerides data needs to be in DATA_DIR and the location must be specified at runtime.
 Thus user's can update them.
+
+
 """
```

But I'd still like to improve the description, as I find it very unclear (and there's a typo and a bunch of blank lines). Maybe we should even completely remove the part about DATA_DIR (if we want to mention this, I would maybe even move it to a module load message)? Users who want to do this probably/hopefully know how to do this, since it's not really properly documented anyway (?).

```suggestion
description = """A suite of C++ libraries for radio astronomy data processing."""
```